code/html/work.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Arbeiten</title>
+  <link rel="stylesheet" href="../style.css" />
+
+</head>
+
+<body>
+  <header>
+
+    <nav id="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </nav>
+
+    <nav id="navbar">
+      <a href="./work.html" class="pagelink">Arbeiten</a>
+      <a href="./about.html">Über uns</a>
+      <a href="./contact.html">Kontakt</a>
+    </nav>
+
+    <a href="../index.html">
+      <div id="logo">
+        salami <br>hoch <br>drei
+      </div>
+    </a>
+
+  </header>
+
+  <h1 class="pagetitle">Arbeiten</h1>
+  <main class="pagecontent">
+    <div class="workpagecontent">
+
+      <div class="workcontainer">
+        <a href="./html-singleworks/kalender2021.html">
+          <div class="workcontainerImage">
+            <img class="workimage" src="../images/adventskalender2020.jpg" alt="Bild des Adventskalenders 2020">
+          </div>
+          <div class="workcontainerText">
+            <h2>Advents&shy;kalender 2021</h2>
+            <p class="workdescription">29.7cm x 42cm<br>Digitaldruck</p>
+            <p class="price">20.-</p>
+            <p class="workdescription">Wir haben noch einige Exemplare, kontaktiere uns gerne für eine Bestellung!</p>
+          </div>
+        </a>
+      </div>
```

Comment:
This is a really nice structure! ⭐ I also like your naming, you have a system that you follow, which is great! Maybe the names would be more easy to follow if you would separate the words with a dash? 😊 Maybe something like this:
`<div class="work-container">`
   `<div class="work-container-image">`